contracts/libraries/YearnCurveHelpers.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "./../interfaces/external/ICurvePool/ICurvePool.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/// @notice Library for Yearn Curve deposit/withdraw
+library YearnCurveHelpers {
+    using SafeERC20 for IERC20;
+
+    /// @dev Get the array of token amount to send to a
+    ///      Curve 2pool to add liquidity
+    /// @param pool The curve 2pool
```

Comment:
Are pool with 2 coins named `2pool` in documentation  or is it an internal naming ?
(same question for other number of coins)

It seems we should review the naming

Reply:
This naming model is not internal, it can be found in other dapps using the Curve/Ellipsis protocol.

Reply:
Alright thank you !